### PR TITLE
Disable debugger module selector when no modules available

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
closes #228

**Comment by loosebazooka** [comment]
@elharo @patflynn

File: core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java [hunk 1]
```diff
@@ -190,11 +190,15 @@ protected ValidationInfo doValidate() {
...
+      return new ValidationInfo(GctBundle.getString("clouddebug.noprojectid"), myElysiumProjectId);
+    }
+
+    if (myDebuggeeTarget.getModel().getSize() == 0) {
+      return new ValidationInfo(GctBundle.getString("clouddebug.nomodulesfound"), myElysiumProjectId);
     }
 
     if (myDebuggeeTarget.getSelectedItem() == null) {
-      return new ValidationInfo(GctBundle.getString("clouddebug.nomodule"));
+      return new ValidationInfo(GctBundle.getString("clouddebug.nomodule"), myDebuggeeTarget);
```
Comment (elharo) [inline core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java]: could we change this to clouddebug.nomoduleselected?

File: core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java [hunk 2]
```diff
@@ -486,7 +494,11 @@ public void run() {
                 public void run() {
                   DebugTarget targetSelection = null;
 
-                  if (debuggees != null && debuggees.getDebuggees() != null) {
+                  if (debuggees == null || debuggees.getDebuggees() == null || debuggees.getDebuggees().size() == 0) {
```
Comment (elharo) [inline core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java]: getDebuggees().isEmpty() (more semantic and perhaps a tad faster when the list isn't empty)

Comment (loosebazooka) [inline core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java]: yup, I'll change that.

**Comment by loosebazooka** [comment]
lets assume we're going the other way for now.

**Pull request closed without merging**